rust/pyo3-geoarrow/src/scalar/mod.rs

```diff
@@ -12,9 +12,9 @@ use geoarrow_expr_geo::util::to_geo::geometry_to_geo;
 use geoarrow_schema::GeoArrowType;
 use geoarrow_schema::error::GeoArrowError;
 use pyo3::exceptions::{PyIOError, PyValueError};
-use pyo3::intern;
 use pyo3::prelude::*;
 use pyo3::types::{PyCapsule, PyTuple};
+use pyo3::{Borrowed, intern};
```

Comment:
ditto, we shouldn't need this because it's already in `prelude`:
https://docs.rs/pyo3/latest/pyo3/prelude/struct.Borrowed.html